src/components/BarlineGraphComponent.jsx

```diff
@@ -1,211 +1,208 @@
-                    },
-                    {
-                        "value": "17000"
-                    },
-                    {
-                        "value": "18000"
-                    },
-                    {
-                        "value": "19000"
-                    },
-                    {
-                        "value": "19000"
-                    },
-                    {
-                        "value": "19000"
-                    },
-                    {
-                        "value": "19000"
-                    },
-                    {
-                        "value": "20000"
-                    },
-                    {
-                        "value": "21000"
-                    },
-                    {
-                        "value": "22000"
-                    },
-                    {
-                        "value": "23000"
-                    }
-                ]
-            },
-            {
-                "seriesname": "Loss incurred",
-                "renderas": "area",
-                "showvalues": "0",
-                "data": [
-                    {
-                        "value": "4000"
-                    },
-                    {
-                        "value": "5000"
-                    },
-                    {
-                        "value": "3000"
-                    },
-                    {
-                        "value": "4000"
-                    },
-                    {
-                        "value": "1000"
-                    },
-                    {
-                        "value": "7000"
-                    },
-                    {
-                        "value": "1000"
-                    },
-                    {
-                        "value": "4000"
-                    },
-                    {
-                        "value": "1000"
-                    },
-                    {
-                        "value": "8000"
-                    },
-                    {
-                        "value": "2000"
-                    },
-                    {
-                        "value": "7000"
-                    }
-                ]
-            }
-        ]
-    }
-    chartConfigs = {
-        id: 'multi_chart',
-        type: 'mscombi2d',
-        width: '45%',
-        height: 400,
-        dataFormat: 'json',
-        dataSource: this.myDataSource,
-    }
-
-    render() {
-        return (
-            <div className="barlineGraph">
-                <Divider hidden />
-                <ReactFC {...this.chartConfigs} />
-            </div>
-        )
-    }
-
-}
+import React from 'react';
+import FusionCharts from 'fusioncharts';
+import Charts from 'fusioncharts/fusioncharts.charts';
+import ReactFC from 'react-fusioncharts';
+import { Divider } from 'semantic-ui-react';
+
+Charts(FusionCharts);
+
+export default class BarlineGraphComponent extends React.Component {
+    myDataSource = {

```

Comment:
The indentation is off. The starting point of `myDataSource` should be 2 spaces from `export`, not 4 spaces. Please move back by 2 spaces. The same comment applies to `chartConfigs` and `render`. Always keep in mind that the indentation for `.js, .jsx, .css` we are adopting is 2 spaces per tab